frameworks/elastic/universe/config.json

```diff
@@ -124,6 +124,11 @@
           "type": "number",
           "default": 1.0
         },
+        "isolation": {
+          "description": "boolean flag to control isolation",
```

Comment:
Would "Enable node isolation" not be a better description?

Reply:
we arent templating this so this will be removed anyway